.travis.yml

```diff
@@ -34,9 +34,21 @@ cache: ccache
 # The default action for Travis-CI is to continue running even if a command fails.
 # See https://github.com/travis-ci/travis-ci/issues/1066.
 # Use the YAML block scalar header (|) to allow easier multiline script coding.
+# Install the desired CMake before running update_deps.py
 
 before_install:
   - set -e
+  - CMAKE_VERSION=3.10.2
+  - |
+    if [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then
+      CMAKE_URL="https://cmake.org/files/v${CMAKE_VERSION%.*}/cmake-${CMAKE_VERSION}-Linux-x86_64.tar.gz"
```

Comment:
Do you know if this is bit-exact with their tarballs on GitHub, where we're sending folks to get the binaries?  If this is a better site, does it have all the package variants, and we should be recommended that instead?

Reply:
Your question made me ask a lot of questions... I'm preparing a design document now.

Yes, to the extent that I've checked, both sites have bitwise identical packages.  But I think the GitHub site is easier for humans to use, so it may make sense to switch.

Reply:
@mikew-lunarg and I reviewed issues and came up with a design document in VulkanTests/design/cmake-ci-version.txt.  The new code follows that plan.